.github/workflows/updatecli.yml

```
@@ -77,9 +77,10 @@ jobs:
         run: ./bin/hermit env -r >> $GITHUB_ENV
       - name: Set Branch
         run: |
-          branchName=$(git branch -r \
-            | grep -E 'origin/[0-9]+\.[0-9]+' \
-            | awk -F/ '{print $2}' \
+          branchName=$(git ls-remote --heads \
```

```suggestion
          set -euo pipefail
          branchName=$(git ls-remote --heads \
```

to fail early in case there is a problem with this command